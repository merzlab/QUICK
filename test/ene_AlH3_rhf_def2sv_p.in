HF  BASIS=def2-sv(p) cutoff=1.0e-9 denserms=1.0e-6  zmake ENERGY DIPOLE CHARGE=0

Al        -2.62270       -0.15651       -0.06051
H         -1.52270       -0.15651       -0.06051
H         -3.17270       -0.86950        0.57127
H         -3.17270        0.55647       -0.69230
