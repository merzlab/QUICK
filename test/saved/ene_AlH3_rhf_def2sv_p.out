  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2021                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **                   This software provided pursuant to a license agreement containing                 **
  **                   restrictions on its disclosure, duplication, and use. This software               **
  **                   contains confidential and proprietary information, and may not be                 **
  **                   extracted or distributed, in whole or in part, for any purpose                    **
  **                   whatsoever, without the express written permission of the authors.                **
  **                   This notice, and the associated author list, must be attached to                  **
  **                   all copies, or extracts, of this software. Any additional                         **
  **                   restrictions set forth in the license agreement also apply to this                **
  **                   software.                                                                         **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Chi, J.; Cruzeiro, V.W.D.; Keipert, K.; Pekurovsky, D.; Mu, D.; Miao, Y.;He, X.; Ayers,K;
  Brothers, E.; GÃ¶tz, A.; Merz,K. M. QUICK-21.03 University of California San Diego, CA and Michigan State
  University, East Lansing, MI, 2021                                                                       
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Fri Mar  5 11:46:56 2021
| INPUT FILE :    ene_AlH3_rhf_def2sv_p.in
| OUTPUT FILE:    ene_AlH3_rhf_def2sv_p.out
| DATE FILE  :    ene_AlH3_rhf_def2sv_p.dat
| BASIS SET PATH: /home/mmanathunga/fix109/basis

 @ Read Job And Atom

  KEYWORD=HF  BASIS=DEF2-SV(P) CUTOFF=1.0E-9 DENSERMS=1.0E-6  ZMAKE ENERGY DIPOLE CHARGE=0                                                                                                                        
 ============== JOB CARD =============
 METHOD = HATREE FOCK
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 Z-MATRIX CONSTRUCTION
 DIPOLE
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE = 1000
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-19
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-09
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = DEF2-SV(P),  TYPE = CARTESIAN
| BASIS FILE = /home/mmanathunga/fix109/basis/DEF2-SV_P.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = AL
 BASIS FUNCTIONS =   19
 For Atom Kind =    2
 ELEMENT = H 
 BASIS FUNCTIONS =    2

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    4    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    3    NUMBER OF NON-HYDROGEN ATOM =    1
 NUMBER OF ELECTRONS     =   16

 -- INPUT GEOMETRY -- :
    AL         -2.6227      -0.1565      -0.0605
    H          -1.5227      -0.1565      -0.0605
    H          -3.1727      -0.8695       0.5713
    H          -3.1727       0.5565      -0.6923

 -- DISTANCE MATRIX -- :
              1           2           3           4
      1     0.00000
      2     2.07870     0.00000
      3     2.07870     3.60041     0.00000
      4     2.07870     3.60041     3.60041     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   25
 NSHELL =   14 NPRIM  =   49
 JSHELL =   14 JBASIS =   30


 @ End Reading Molecular Information 


 @ Begin Energy Calculation

                                         SCF ENERGY
------------------------------------------------------------------------------------------------------------------------
NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
------------------------------------------------------------------------------------------------------------------------
|  1   -243.836417469      ------         0.023     1        0.00      0.02      0.00    0.2574E+01  0.6524E-01  0.4615E+00
|  2   -243.137322694  -.699095E+00       0.024     2        0.00      0.02      0.00    0.5279E-01  0.1321E-01  0.1371E+00
|  3   -243.154887042  0.175643E-01       0.024     3        0.00      0.02      0.00    0.2317E-01  0.4983E-02  0.5023E-01
|  4   -243.157296385  0.240934E-02       0.024     4        0.00      0.02      0.00    0.8983E-02  0.1365E-02  0.7212E-02
|  5   -243.157595444  0.299059E-03       0.024     5        0.00      0.02      0.00    0.2319E-02  0.3954E-03  0.1963E-02
|  6   -243.157615966  0.205217E-04       0.024     6        0.00      0.02      0.00    0.1657E-03  0.6049E-04  0.4014E-03
|  7   -243.157616137  0.170724E-06       0.024     7        0.00      0.02      0.00    0.1711E-04  0.8863E-05  0.5021E-04
|  8   -243.157616139  0.239685E-08       0.024     8        0.00      0.02      0.00    0.3107E-05  0.1422E-05  0.7973E-05
|  9   -243.157616139  0.641656E-10       0.024     9        0.00      0.02      0.00    0.2381E-06  0.5385E-07  0.2861E-06
------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   9 CYLCES
| MAX ERROR = 0.238150E-06   RMS CHANGE = 0.538450E-07   MAX CHANGE = 0.286094E-06
 -----------------------------------------------
 ELECTRONIC ENERGY    =  -262.752592104
 CORE_CORE REPULSION  =    19.594975965
 TOTAL ENERGY         =  -243.157616139

 @ End Energy calculation


 @ Begin Zmake conversion

  
 Z-MATRIX:

   ATOM  ELEMENTAL  BOND LENGTH    BOND ANGLE    DIHEDRAL ANGLE
  NUMBER  SYMBOL    (ANGSTROMS)    (DEGREES)       (DEGREES)
     I                 I-NA         I-NA-NB        I-NA-NB-NC      NA   NB   NC

     1      AL       0.00000  0     0.00000  0      0.00000  0      0    0    0
     2      H        1.10000  1     0.00000  0      0.00000  0      1    0    0
     3      H        1.10000  1   119.99999  1      0.00000  0      1    2    0
     4      H        1.10000  1   120.00001  1   -179.99915  1      1    2    3

 @ End Zmake conversion


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   AL             0.5565             0.1699
   H             -0.1855            -0.0687
   H             -0.1855            -0.0506
   H             -0.1855            -0.0506
   TOTAL        -0.0000            -0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
   -0.0000    0.0000    0.0000    0.0000

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     3.273771000( 93.64%)
| TOTAL SCF TIME      =     0.215192000(  6.16%)
|       TOTAL OP TIME      =     0.213834000(  6.12%)
|             TOTAL 1e TIME      =     0.001124000(  0.03%)
|             TOTAL 2e TIME      =     0.211477000(  6.05%)
|       TOTAL DII TIME      =     0.001296000(  0.04%)
|             TOTAL DIAG TIME    =     0.000529000(  0.02%)
| TOTAL TIME          =     3.496130000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes  3.5 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Fri Mar  5 11:46:59 2021
